docker_daemon/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [IMPROVEMENT] Add custom tags to all service checks. See [#782][]
 * [IMPROVEMENT] Add docker memory soft limit metric. See [#760][]
 * [IMPROVEMENT] Add docker.containers.running.total & docker.containers.stopped.total metrics. See [#859][]
+* [FEATURE] Add COLLECT_LABELS_AS_TAGS as an environment variable. See [#881][]
```

Comment:
I'd say `honor global collect_labels_as_tags if integration's collect_labels_as_tags is empty` instead of talking about envvars. Envvars is just a side effect